### DISCUSSION TITLE
Footer - Previous, Next

### DISCUSSION THREAD

**Discussion by JesseWebDotCom** [mkdocstrings/mkdocstrings#257] [Q&A] [answered]
On your documentation site I see a footer with Previous / Next navigation but I can't seem to get that to appear.  Is there anything special I need to do?  Also, I didn't see a template file for it in templates/python/material

**Comment by oprypin** [comment, marked as answer]
This is a MkDocs feature, relying on having every page in the nav. Without a repro case from you, I can only guess that you didn't define a nav.
